containers/aesmd/Dockerfile

```diff
@@ -19,24 +19,9 @@ RUN echo 'deb [arch=amd64] https://packages.microsoft.com/ubuntu/20.04/prod foca
 
 RUN apt-get update
 
-# TODO: Update Conclave to Intel SGX SDK 2.18 and remove package versions: https://r3-cev.atlassian.net/browse/CON-1243
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    sgx-aesm-service=2.17.100.3-focal1 \
-    libsgx-ae-id-enclave=1.14.100.3-focal1 \
-    libsgx-ae-le=2.17.100.3-focal1 \
-    libsgx-ae-pce=2.17.100.3-focal1 \
-    libsgx-ae-qe3=1.14.100.3-focal1 \
-    libsgx-ae-qve=1.14.100.3-focal1 \
-    libsgx-aesm-ecdsa-plugin=2.17.100.3-focal1 \
-    libsgx-aesm-launch-plugin=2.17.100.3-focal1 \
-    libsgx-aesm-pce-plugin=2.17.100.3-focal1 \
-    libsgx-aesm-quote-ex-plugin=2.17.100.3-focal1 \
-    libsgx-dcap-quote-verify=1.14.100.3-focal1 \
-    libsgx-enclave-common=2.17.100.3-focal1 \
-    libsgx-launch=2.17.100.3-focal1 \
-    libsgx-pce-logic=1.14.100.3-focal1 \
-    libsgx-qe3-logic=1.14.100.3-focal1 \
-    libsgx-urts=2.17.100.3-focal1 \
+    libsgx-quote-ex \
```

Comment:
Does one of these packages also installed the aesm service?

Reply:
I am following the instructions from Intel
```
To configure the system to run an Intel® SGX application:
1. Install the following Debian Library and App Packages: libsgx-epid, libsgx-quote-
ex, libsgx-dcap-ql, which also installs the dependent packages libsgx-urts, libsgx-
launch, libsgx-ae-le, libsgx-ae-pce, libsgx-ae-qe3, libsgx-ae-qve, libsgx-
ae-epid, libsgx-qe3-logic, libsgx-pce-logic, libsgx-dcap_quote-verify,
libsgx-aesm-ecdsa-plugin, libsgx-aesm-epid-plugin, libsgx-ae-launch-
plugin, libsgx-aesm-quote-ex-plugin, libsgx-enclave-common, and sgx-aesm-
service following these steps
```
...
```
Update apt and install the following packages:
sudo apt-get update
sudo apt-get install libsgx-epid libsgx-quote-ex libsgx-dcap-ql
```
I have removed the epid package here.

Interestingly though, when using some tools, it looks like Ubuntu is installing packages which are actually not strictly "necessary", but "recommended". This is something that has been enabled by default for 10 years or so.

But, anyway, when doing
```
sudo apt install libsgx-dcap-ql libsgx-quote-ex --dry-run
```
this happens

```
The following NEW packages will be installed:
  libsgx-ae-epid libsgx-ae-id-enclave libsgx-ae-le libsgx-ae-pce libsgx-ae-qe3 libsgx-ae-qve libsgx-aesm-ecdsa-plugin libsgx-aesm-epid-plugin libsgx-aesm-launch-plugin libsgx-aesm-pce-plugin
  libsgx-aesm-quote-ex-plugin libsgx-dcap-ql libsgx-dcap-quote-verify libsgx-enclave-common libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex libsgx-urts
```

Note that if we installed those two packages one by one, some dependencies would overlap

Reply:
Can we try to exclude the epid dependencies?

Reply:
I feel it's a bit overshooting, but it does not harm, so I will just remove the versions from the previous version of the file.